modules/alma/lib/AlmaClient/AlmaClient.class.php

```diff
@@ -438,7 +446,7 @@ public function add_reservation($borr_card, $pin_code, $reservation) {
     $params = array(
       'borrCard' => $borr_card,
       'pinCode' => $pin_code,
-      'reservable' => rawurlencode($reservation['id']),
+      'reservable' => $reservation['id'],
```

Comment:
Why is this changed? This is required for [issue 1337](http://platform.dandigbib.org/issues/1337) and included in ding2/alma#21?